crossword-xml-uploader/src/main/scala/com/gu/crossword/Lambda.scala

```diff
@@ -27,6 +27,27 @@ trait CrosswordUploaderLambda
     }
   }
 
+  // Upload to crosswordv2 service - do NOT createPage
+  // This should be removed once the crosswordv2 service is ready to be used
+  // Wrapping with Try as we must fail safe and not stop the lambda from running if this fails
+  private def doV2Upload(url: String, crosswordXmlFile: CrosswordXmlFile): Unit = Try {
+    println(s"Attempting to dual upload crossword ${crosswordXmlFile.key} to crosswordv2")
+    val v2Result = for {
+      rawXml <- uploadCrossword(url)(crosswordXmlFile)
+      _ <- XmlProcessor.process(rawXml)
```

Comment:
it's occurring to me belatedly, but if we're dual running the two apps, then there's much less pressure to be exactly compatible. is it worth taking the opportunity to swap out of the odd XML return format; either completely aligning with what flexible-integration expects, or responding with JSON from crosswordv2? (I'd lean towards the former personally; avoids needing to pull in a JSON lib)

Reply:
I'm in favour of that. Let's get this working as is, then we can iterate knowing we aren't going to break the existing workflow.